manager/api/es/src/main/java/io/apiman/manager/api/es/EsStorage.java

```diff
@@ -2308,6 +2210,14 @@ public ApiVersionBean unmarshal(Map<String, Object> source) {
         }, qb);
     }
 
+    /**
+     * Refreshes an index
+     * @param indexPostfix the index postfix name
+     */
+    public void refresh(String indexPostfix) throws IOException {
```

Comment:
Is this method still in use @bekihm ?

Reply:
Yes it is used for the tests from classes like TestEsStorageWrapper and TestEsStorageQueryWrapper